client/.storybook/addons.js

```diff
@@ -1,4 +1,4 @@
-import "@storybook/addon-actions/register";
-import "@storybook/addon-links/register";
-import "@storybook/addon-knobs";
-import "@storybook/addon-viewport";
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-viewport/register';
```

Comment:
that's the prettifying issue Leon talked about, but there are not many files left, so all good

Reply:
If you prefer double quotes on across your code -> change the .prettierrc to do so. 
This still lgtm though.